cuda/extensions.bzl

```diff
@@ -74,6 +78,40 @@ cuda_toolkit_tag = tag_class(attrs = {
     ),
 })
 
+platform_alias_tag = tag_class(
+    attrs = {
+        "name": attr.string(
+            mandatory = True,
+            doc = "Name of the alias repository to create",
+        ),
+        "component_name": attr.string(
+            mandatory = True,
+            doc = "Name of the component to create aliases for",
+        ),
+        "linux_x86_64_repo": attr.string(
+            mandatory = True,
+            doc = "Name of the repository to use for x86_64 platform",
+        ),
+        "linux_aarch64_repo": attr.string(
+            mandatory = True,
+            doc = "Name of the repository to use for ARM64/Jetpack platform",
+        ),
+        "linux_sbsa_repo": attr.string(
+            mandatory = True,
+            doc = "Name of the repository to use for SBSA platform",
+        ),
```

Comment:
would a `platform_mapping` with `attr.string_dict` be better here? Validating the `key`  in the rule impl seem to be future proof.